app/models/qdm/tacoma/measure.rb

```diff
@@ -64,34 +58,29 @@ class Measure
     field :populations_cql_map, type: Hash
     field :observations, type: Array
 
-    field :value_set_oids, type: Array, default: []
-    field :value_set_oid_version_objects, type: Array, default: []
-
-    field :complexity, type: Hash
-
     # Relations to other model classes
-    belongs_to :user
-    belongs_to :bundle, class_name: 'HealthDataStandards::CQM::Bundle'
-    has_and_belongs_to_many :patients, inverse_of: nil
-    has_one :package, class_name: 'CqlMeasurePackage', inverse_of: :measure, dependent: :destroy
+    belongs_to :bundle, class_name: 'HealthDataStandards::CQM::Bundle' # Cypress-specific, until we migrate the Bundle into cqm-models
+    has_one :package, class_name: 'CqlMeasurePackage', inverse_of: :measure, dependent: :destroy # Bonnie-specific
+    has_and_belongs_to_many :patients
+
+    # Store this references as an Array on the Measure object,
+    # but don't care about the inverse relationship (e.g. we never really care 
+    # about getting all the measures from a ValueSet object,
+    # hence the 'inverse_of: nil')
+    has_and_belongs_to_many :value_sets, inverse_of: nil
 
     scope :by_measure_id, ->(id) { where('measure_id' => id) }
     scope :by_type, ->(type) { where('type' => type) }
     scope :by_user, ->(user) { where user_id: user.id }
 
     index 'user_id' => 1
+    index 'hqmf_id' => 1
```

Comment:
`hqmf_id` wouldn't be a useful index.

A compound index on `user_id` and `hqmf_set_id` would be useful since it's the most common query in bonnie.
`index 'user_id' => 1, 'hqmf_set_id' => 1`